src/components/jurisdiction/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@bcrs-shared-components/jurisdiction",
+  "version": "1.0.0",
```

Comment:
I think the version here should be 0.0.0 Leo. 

When we do lerna publish, it should automatically make it 1.0.0!

Reply:
LGTM. OK to resolve?

Reply:
Fixed, thanks! Resolving.